UnitsNet/GeneratedCode/Extensions/Number/NumberToPowerDensityExtensions.g.cs

```diff
@@ -1132,6 +1166,40 @@ public static class NumberToPowerDensityExtensions
 
         #endregion
 
+        #region PetawattPerCubicMeter
+
+        /// <inheritdoc cref="PowerDensity.FromPetawattsPerCubicMeter(UnitsNet.QuantityValue)" />
+        public static PowerDensity PetawattsPerCubicMeter(this int value) => PowerDensity.FromPetawattsPerCubicMeter(value);
```

Comment:
Consider moving the PowerDensity units to a separate PR, even if that is probably why you started working on the abs vs relative comparison to begin with.

Reply:
That is indeed why I started :) I just forgot to remove it. Good catch